build.psm1

```diff
@@ -1854,12 +1800,13 @@ function Start-TypeGen
     <Target Name="_GetDependencies"
             DependsOnTargets="ResolveAssemblyReferencesDesignTime">
         <ItemGroup>
-            <_RefAssemblyPath Include="%(_ReferencesFromRAR.HintPath)%3B"  Condition=" '%(_ReferencesFromRAR.NuGetPackageId)' != 'Microsoft.Management.Infrastructure' "/>
+            <_RefAssemblyPath Include="%(_ReferencesFromRAR.OriginalItemSpec)%3B" Condition=" '%(_ReferencesFromRAR.IsSystemReference)' == 'true' and '%(_ReferencesFromRAR.NuGetPackageId)' != 'Microsoft.Management.Infrastructure' "/>
```

Comment:
Same as above comment, the `.inc` file produced doesn't contain the ref assemblies for other nuget packages we reference.

Reply:
Will remove check for systemreference

Reply:
Right, I just inspected the build log, removing `'%(_ReferencesFromRAR.IsSystemReference)' == 'true' and` would fix it.